Unit test fails on python 3.14

**Describe the bug**
unit test fails on py3.14
```
FAILED tests/test_prefseditor_keybindings.py::test_duplicate_accels_not_possible_to_set[accel_params0] - assert (97, <flags GDK_CONTROL_MASK | GDK_MOD1_MASK of type Gdk.ModifierType>) in {(accelerator_key=43, accelerator_mods=<flags GDK_CONTROL_MASK of type Gdk.ModifierType>), (accelerator_key=45, accelerator_mods=<flags GDK_CONTROL_MASK of type Gdk.ModifierType>), (accel...
FAILED tests/test_prefseditor_keybindings.py::test_duplicate_accels_not_possible_to_set[accel_params1] - assert (97, <flags GDK_CONTROL_MASK | GDK_MOD1_MASK of type Gdk.ModifierType>) in {(accelerator_key=43, accelerator_mods=<flags GDK_CONTROL_MASK of type Gdk.ModifierType>), (accelerator_key=45, accelerator_mods=<flags GDK_CONTROL_MASK of type Gdk.ModifierType>), (accel...
FAILED tests/test_prefseditor_keybindings.py::test_duplicate_accels_not_possible_to_set[accel_params2] - assert (65480, <flags 0 of type Gdk.ModifierType>) in {(accelerator_key=43, accelerator_mods=<flags GDK_CONTROL_MASK of type Gdk.ModifierType>), (accelerator_key=45, accelerator_mods=<flags GDK_CONTROL_MASK of type Gdk.ModifierType>), (accelerator_key=65289, accelerato...
FAILED tests/test_prefseditor_keybindings.py::test_duplicate_accels_not_possible_to_set[accel_params3] - assert (122, <flags GDK_SHIFT_MASK | GDK_CONTROL_MASK of type Gdk.ModifierType>) in {(accelerator_key=43, accelerator_mods=<flags GDK_CONTROL_MASK of type Gdk.ModifierType>), (accelerator_key=45, accelerator_mods=<flags GDK_CONTROL_MASK of type Gdk.ModifierType>), (acc...
```

**To Reproduce**
pytest in terminator's source

**Expected behavior**
test should pass

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**
 - OS or Linux Distribution: [e.g. FreeBSD or Ubuntu] Gentoo linux, amd64
 - Version [e.g. 20.04]
 - Display Technology: X11


**Additional context**
Add any other context about the problem here.

log: [build.txt](https://github.com/user-attachments/files/27978220/build.txt)

--- a/tests/test_prefseditor_keybindings.py
+++ b/tests/test_prefseditor_keybindings.py
@@ -56,7 +56,7 @@ def test_non_empty_default_keybinding_accels_are_distinct():
     from terminatorlib import config
 
     all_default_accelerators = [
-        Gtk.accelerator_parse(accel)
+        tuple(Gtk.accelerator_parse(accel))
         for accel in config.DEFAULTS["keybindings"].values()
         if accel != ""  # ignore empty key bindings
     ]
@@ -156,7 +156,7 @@ def test_duplicate_accels_not_possible_to_set(accel_params):
     binding = liststore.get_value(liststore.get_iter(path), 0)
 
     all_default_accelerators = {
-        Gtk.accelerator_parse(accel)
+        tuple(Gtk.accelerator_parse(accel))
         for accel in config.DEFAULTS["keybindings"].values()
         if accel != ""  # ignore empty key bindings
     }

